docs/content/installation/install_compatibility_requirements.md

```diff
@@ -49,9 +49,9 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
    `oc apply -f 99-ibm-attach.yaml`
 
-2. Configure your storage system host attachment, per worker node.
+2. (Optional): Configure your storage system host attachment, per worker node.
```

Comment:
it's not optional by default, because installing the host definer is optional by itself (i.e. they cannot both be optional independently, one of them has to be done)

Reply:
updated